### DISCUSSION TITLE
Issues with inference

### DISCUSSION THREAD

**Discussion by mitzmit** [talmolab/sleap#1796] [Help!]
Hello!
I am using the Colab template for sleap with slight changes for my own file names (sc attached)
<img width="347" alt="Screenshot 2024-06-04 at 5 40 01 PM" src="https://github.com/talmolab/sleap/assets/131608486/c5a6008a-697a-4ed8-a2c5-3829f3d80b49">

However, I get the following error:

<img width="814" alt="Screenshot 2024-06-04 at 5 38 53 PM" src="https://github.com/talmolab/sleap/assets/131608486/ac8f9f59-9179-4776-a612-3d41769eecd9">

I have no clue what could be causing this or how to fix this-- Any advice??

**Comment by eberrigan** [comment]
Hi @mitzmit,

Could you upload your sleap package and model here and I will try it on my computer/environment? 
[Google Drive Upload](https://docs.google.com/forms/d/e/1FAIpQLSfWBwcuZH8lxGyFabMAODLjSUE4YrSCYbrrnYvXCpgBqZpcJA/viewform)
Let me know if they are too big.

Thanks!

Elizabeth

**Comment by mitzmit** [comment]
Hello! It was too big—so sorry for delay this email got lost in my inbox—How should I send it to you?

Thanks so much.

Jacqueline
From: Elizabeth ***@***.***>
Date: Friday, June 7, 2024 at 12:13 PM
To: talmolab/sleap ***@***.***>
Cc: Jacqueline R Mitzenmacher ***@***.***>, Mention ***@***.***>
Subject: Re: [talmolab/sleap] Issues with inference (Discussion #1796)

Hi @mitzmit<https://github.com/mitzmit>,

Could you upload your sleap package and model here and I will try it on my computer/environment?
Google Drive Upload<https://docs.google.com/forms/d/e/1FAIpQLSfWBwcuZH8lxGyFabMAODLjSUE4YrSCYbrrnYvXCpgBqZpcJA/viewform>
Let me know if they are too big.

Thanks!

Elizabeth

**Comment by talmo** [comment]
Hi @mitzmit,

This may have to do with the current issues with Google Colab support -- see #1644 for a description and possible workarounds.

Let us know if we can help!

Talmo
